lib/zmq-kernel.js

```diff
@@ -429,7 +429,7 @@ export default class ZMQKernel extends Kernel {
 
     if (this.kernelProcess) {
       this._kill();
-      fs.unlink(this.connectionFile);
+      fs.unlinkSync(this.connectionFile);
```

Comment:
I'm wondering if this could potentially freeze the UI if `unlinkSync` takes longer than normal, since it is now synchronous?

I didn't know that calling async functions without a callback was deprecated.
@t9md Should we maybe just add a pseudo callback instead?